.github/workflows/dev-pr.yml

```diff
@@ -18,37 +21,48 @@ jobs:
     if: contains(fromJson('["workflow_dispatch"]'), github.event_name) || contains(github.event.pull_request.labels.*.name, 'deploy to dev')
     runs-on: ubuntu-latest
     container: node:14
+    permissions:
+        pull-requests: write
     env:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      PR_NAME: PR${{ github.event.pull_request.number }}
+      PR_NAME: PR-${{ github.event.pull_request.number || github.event.inputs.pr_number || 'workflow-dispatch' }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: install
         run: npm ci
       - name: clear S3
-        run: node tools/scripts/delete-s3-dir.js \
+        run: |
+          node ./tools/scripts/delete-s3-dir.js \
             --bucket ${{ secrets.AWS_BUCKET }} \
             --role ${{ secrets.AWS_ROLE_ARN }} \
-            --pr PR${{ github.event.pull_request.number }}
+            --pr PR-${{ github.event.pull_request.number || github.event.inputs.pr_number || 'workflow-dispatch' }}
       - name: build
         if: github.event.inputs.pr_build == 'build' || github.event.action != 'closed'
```

Comment:
Did you mean to use `&&` rather than `||` ?

Reply:
> Did you mean to use `&&` rather than `||` ?

Yes thanks nice catch